src/components/TVSet.vue

```diff
@@ -74,7 +74,7 @@ img {
 }
 
 #tv {
-  transform: translate3d(20px, -233px, -50px);
+  transform: translate3d(33px, -233px, -50px);
```

Comment:
This changes seems unnecessary. It looks slightly better with 20px in my eyes.

Reply:
The problem was the TV set to protrude into the wall. Actually, this should not really be a problem because nobody will notice, as long as one cannot move out of the room. I only noticed it while experimenting with the archive room and changing the camera position.
I will change that back.